src/TodoInput.jsx

```diff
@@ -0,0 +1,10 @@
+export default function TodoInput({ text, onChange, onClick }) {
+  return (
+    <>
+      <div>
+        <input type="text" placeholder="할 일을 입력해주세요" value={text} onChange={onChange} />
```

Comment:
```suggestion
        <input 
          type="text"
          placeholder="할 일을 입력해주세요"
          value={text}
          onChange={onChange}
        />
```

저는 속성이 2개 이상일 때는 줄바꿈해주면 코드가독성이 더 좋다고 생각하고 코드를 정리해주는 편이에요.
코드 가독성을 고려한 자기만의 코드 정리 기준을 세우시면 좋아요.

Reply:
자기만의 코드정리 기준.. 고민해보겠습니다